docs/src/how-to/administrate/users.rst

```diff
@@ -607,3 +607,40 @@ Where:
 * `$EMAIL_CODE` is the validation code received by email after running the previous script/command
 * `$TEAM_CURRENCY` is the currency of the team
 * `$TEAM_NAME` is the name of the team
+
+Enabling debugging for backend services
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, backend services (pods) might not output as much information as needed to resolve specific issues.
+
+You can change this for any of the services by editing its configuration.
+
+For example, to enable debugging for the `cargohold` service, you can edit the `cargohold` configuration in your ``values.yaml``:
```

Comment:
*debugging* is a term that's usually used differently in software engineering (`gdb`, break points, *DWARF* debug symbols, ...). I would refer to *debug logs* or better *log level debug*.

```suggestion
For example, to enable debug logs for the `cargohold` service, you can edit the `cargohold` configuration in your ``values.yaml``:
```